pkgs/applications/office/espanso/default.nix

```diff
@@ -3,67 +3,116 @@
 , rustPlatform
 , pkg-config
 , extra-cmake-modules
+, dbus
 , libX11
 , libXi
 , libXtst
 , libnotify
+, libxkbcommon
 , openssl
 , xclip
 , xdotool
+, setxkbmap
+, wl-clipboard
+, wxGTK32
 , makeWrapper
 , stdenv
 , AppKit
 , Cocoa
 , Foundation
+, IOKit
+, Kernel
+, waylandSupport ? false
```

Comment:
```suggestion
, enableWayland ? stdenv.isLinux
, enableX11 ? stdenv.isLinux
```

Reply:
I've gone for x11Support, as there seemed to be more cases where this was used already.